desktop/packages/windows/wix/ruffle.wxs

```diff
@@ -0,0 +1,182 @@
+                what they want the next time they open a swf.
+                -->
+
+            <ProgId Id='Ruffle.swf' Description='Flash Movie' Icon="DesktopEXE" IconIndex="1">
+                <Extension Id='swf' ContentType='application/x-shockwave-flash'>
+                    <Verb Id='open' Command='Open' TargetFile='DesktopEXE' Argument='"%1"'/>
+                </Extension>
+            </ProgId>
+            <ProgId Id='Ruffle.spl' Description='Flash Movie' Icon="DesktopEXE" IconIndex="1">
+                <Extension Id='spl' ContentType='application/x-shockwave-flash'>
+                    <Verb Id='open' Command='Open' TargetFile='DesktopEXE' Argument='"%1"'/>
+                </Extension>
+            </ProgId>
+            <ProgId Id='Ruffle.ruf' Description='Ruffle Bundle' Icon="DesktopEXE" IconIndex="1">
+                <Extension Id='ruf' ContentType='application/x-ruffle-bundle'>
+                    <Verb Id='open' Command='Open' TargetFile='DesktopEXE' Argument='"%1"'/>
+                </Extension>
+            </ProgId>
+
+            <!-- This tells Windows to include Ruffle in the "Open With" dialog -->
+            <RegistryValue Root="HKCR" Key=".swf\OpenWithProgids" Name="Ruffle.swf" Value="" Type="string"/>
+            <RegistryValue Root="HKCR" Key=".spl\OpenWithProgids" Name="Ruffle.spl" Value="" Type="string"/>
+            <RegistryValue Root="HKCR" Key=".ruf\OpenWithProgids" Name="Ruffle.ruf" Value="" Type="string"/>
+        </Component>
+
+        <Component Id='ApplicationInfo' Guid='f7e7c5bb-729c-427e-b913-fd0d2e11a143'>
+            <!--
+                Tell windows a few properties about our application, so that it can handle it slightly better in "Open With"
+                
+                (wondering how Windows matches these to the ruffle.exe that we install at some arbitrary path the user picks?)
+                (well that's the fun part... it doesn't! Windows SOLELY goes by the filename. Isn't that convenient?!)
+                (rename an executable and suddenly windows can't associate these things with it anymore whoops)
+            -->
+
+            <RegistryKey Root="HKCR" Key="Applications\ruffle.exe">
+                <!-- Give Ruffle a better name in the "Open With" dialog, than just "ruffle.exe" -->
+                <RegistryValue Name="FriendlyAppName" Value="Ruffle" Type="string"/>
+
+                <!-- Hint that we can open certain files -->
+                <RegistryValue Key="SupportedTypes" Name=".swf" Value="" Type="string"/>
+                <RegistryValue Key="SupportedTypes" Name=".spl" Value="" Type="string"/>
+                <RegistryValue Key="SupportedTypes" Name=".ruf" Value="" Type="string"/>
+            </RegistryKey>
+        </Component>
+
+        <Feature
+                Id='Binaries'
+                Title='Application'
+                Description='Installs the Ruffle desktop player'
```

Comment:
Should it be "player" here? Maybe just "application"? I thought that it was decided that Ruffle should be referred to as an emulator :sweat_smile: